local-cli/core/getCommands.js

```diff
@@ -2,11 +2,24 @@ const path = require('path');
 const findPlugins = require('./findPlugins');
 const flatten = require('lodash').flatten;
 
+const attachPackage = (command, pkg) => Array.isArray(command)
+  ? command.map(cmd => attachPackage(cmd, pkg))
+  : { ...command, pkg };
+
 /**
  * @return {Array} Array of commands
  */
 module.exports = function getCommands() {
   const appRoot = process.cwd();
-  const plugins = findPlugins([appRoot]).map(name => require(path.join(appRoot, 'node_modules', name)));
+  const plugins = findPlugins([appRoot])
+    .map(pathToCommands => {
```

Comment:
pathToCommands can be `react-native-windows/cli/commands.js` - since we want to read the package.json we need to split. 

Note, scoped modules are not considered since all plugins here have to follow strict naming conventions (either `rnpm-plugin-*` or `react-native-*`).